server/src/routers/converterRouter.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+const multer = require('multer');
+const upload = multer();
+const router = new express.Router();
+
+const { mapping } = require('../constants/config');
+const { parseInputFile, sendResponse } = require('../util');
+
+router.use(express.json());
+
+var Converter = require('api-spec-converter');
+
+//Endpoint for conversion
```

Comment:
Is this necessary? Remove unwanted comments, make your endpoint readable if it's not. No need for these comments.